core/controllers/ingest.py

```diff
@@ -245,47 +244,46 @@ def extract_data(excel_file: FileStorage) -> dict[str, pd.DataFrame]:
     return workbook
 
 
-def clean_data(data: dict[str, pd.DataFrame]) -> None:
-    """Clean the data in the given workbook by replacing all occurrences of `np.NA` with an empty string and `np.nan`
+def clean_data(transformed_data: dict[str, pd.DataFrame]) -> None:
+    """Clean the transformed data by replacing all occurrences of `np.nan` with an empty string and `pd.NaT`
     with None.
 
     :param data: A dictionary where the keys are table names and the values are pd.DataFrames
     :return: None
     """
-    for table in data.values():
-        table.fillna("", inplace=True)  # broad replace of np.NA with empty string
-        table.replace({np.nan: None}, inplace=True)  # replaces np.NAT with None
+    for table in transformed_data.values():
+        table.fillna("", inplace=True)
+        # fillna with empty string does not work for datetime columns, so we need to replace NaT with None
+        table.replace({pd.NaT: None}, inplace=True)
```

Comment:
Can we add a unit test for ```clean_data``` with an example of pd.NaT in? Can we also make sure this test has an example of np.nan in to make sure that functionality still works (if such a test doesn't already exist)?